app/controllers/application_controller.rb

```diff
@@ -6,13 +6,11 @@ class ApplicationController < ActionController::Base
   before_action :clear_association_queries
   around_action :configure_time_machine
 
-  unless Rails.application.config.consider_all_requests_local
```

Comment:
I think this will confuse matters in development mode and prevent us from seeing the normal rails error message

Reply:
Isn't this `unless` to make sure these exceptions are not rescued in dev mode, so falling through to the default rails exception handler app

Without it, the dev mode will operate the same as prod mode without the red exceptions screens?